src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ticketRuleModal.tsx

```diff
@@ -73,158 +106,110 @@ class TicketRuleModal extends React.Component<Props, State> {
+      {
+        name: 'title',
+        label: 'Title',
+        type: 'string',
+        default: 'This will be the same as the Sentry Issue.',
+        disabled: true,
+      } as IssueConfigField,
+      {
+        name: 'description',
+        label: 'Description',
+        type: 'string',
+        default: 'This will be generated from the Sentry Issue details.',
+        disabled: true,
+      } as IssueConfigField,
+    ];
+
+    const configsFromAPI = (integrationDetails || {})[this.getConfigName()];
+    return fields.concat(
+      (configsFromAPI || [])
+        // Skip fields if they already exist.
+        .filter(field => !fields.map(f => f.name).includes(field.name))
+        .map(field => {
+          // Overwrite defaults from cache.
+          if (instance.hasOwnProperty(field.name)) {
+            field.default = instance[field.name] || field.default;
+          }
+
+          // Overwrite choices from cache.
+          if (fetchedFieldOptionsCache?.hasOwnProperty(field.name)) {
+            field.choices = fetchedFieldOptionsCache[field.name];
+          }
+
+          return field;
+        })
     );
+  };
 
-    const submitLabel = t('Apply Changes');
-    const cancelLabel = t('Close');
-    const fields = Object.values(formFields);
-    this.addFields(fields);
-    const initialData = instance || {};
-    fields.forEach((field: FormField) => {
-      // passing an empty array breaks multi select
-      // TODO(jess): figure out why this is breaking and fix
-      if (!initialData.hasOwnProperty(field.name)) {
-        initialData[field.name] = field.multiple ? '' : field.default;
-      }
-    });
-
+  renderBodyText = () => {
+    const {ticketType, link} = this.props;
     return (
-      <React.Fragment>
-        <Header closeButton>{t('Issue Link Settings')}</Header>
-        <Body>
-          <BodyText>
-            {text}
-            {tct("It'll also [linkToDocs] with the new Sentry Issue.", {
-              linkToDocs: <ExternalLink href={link}>{t('stay in sync')}</ExternalLink>,
-            })}
-          </BodyText>
-          <Form
-            onSubmit={this.onFormSubmit}
-            initialData={initialData}
-            submitLabel={submitLabel}
-            cancelLabel={cancelLabel}
-            footerClass="modal-footer"
-            onCancel={closeModal}
-          >
-            {fields
-              .filter((field: FormField) => field.hasOwnProperty('name'))
-              .map((field: IssueConfigField) => (
-                <FieldFromConfig
-                  key={`${field.name}-${field.default}-${field.required}`}
-                  field={field}
-                  inline={false}
-                  stacked
-                  flexibleControlStateSize
-                  {...this.getFieldProps(field)}
-                />
-              ))}
-          </Form>
-        </Body>
-      </React.Fragment>
+      <BodyText>
+        {tct(
+          'When this alert is triggered a {ticketType} will be ' +
```

Comment:
@mgaeta I think it needs to be wrapped in square brackets
```suggestion
          'When this alert is triggered a [ticketType] will be ' +
```

Reply:
good catch